pkg/flowaggregator/flowaggregator.go

```diff
@@ -215,7 +324,7 @@ func (fa *flowAggregator) sendFlowKeyRecords(key ipfixintermediate.FlowKey, reco
 			return fmt.Errorf("AddRecord failed, error: %v", err)
 		}
 	}
-	bytesSent, err := fa.exportingProcess.AddSetAndSendMsg(ipfixentities.Data, setSent.GetSet())
+	bytesSent, err := fa.exportingProcess.SendSet(setSent.GetSet())
```

Comment:
Do we still need the function sendFlowKeyRecords? It is not used anywhere.

Reply:
It's not needed. Removed. Thanks!